### PR TITLE
Update comment.md

### PR DESCRIPTION
**Pull request by Wabri**
# Description

Wrong code example in readme of filter_lines comment filter 

## Issues Resolved

Nope

## Check List

- [x] All tests pass. See TESTING.md for details.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by xorima** [comment]
@Wabri 
Can you update the `CHANGELOG.md` please with an entry so we can get this merged?

**Review by xorima — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## 4.4.1 - *2021-09-21*
 
+- Documentation fixes for filter_lines comment filter
```
Comment (xorima) [inline CHANGELOG.md]:
```suggestion
- Documentation fixes for filter_lines comment filter

## 4.4.1 - *2021-09-21*

```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,9 +5,11 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+- Documentation fixes for filter_lines comment filter
```
Comment (xorima) [inline CHANGELOG.md:9]:
```suggestion

- Documentation fixes for filter_lines comment filter
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,8 +6,11 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## Unreleased
 
+- Documentation fixes for filter_lines comment filter
+
 ## 4.4.1 - *2021-09-21*
 
+
```
Comment (xorima) [inline CHANGELOG.md]:
```suggestion
```

**Comment by xorima** [comment]
Sorry for noise, just moving your changelog entry to the right spot

**Comment by Wabri** [comment]
Sorry for the missunderstanding and thank you

**Review by xorima — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [4.4.2](https://github.com/sous-chefs/line/releases/tag/4.4.2)

**Pull request merged**